src/pip/_internal/index/collector.py

```diff
@@ -434,7 +434,7 @@ def _get_html_page(link, session=None):
     # Check for VCS schemes that do not support lookup as web pages.
     vcs_scheme = _match_vcs_scheme(url)
     if vcs_scheme:
-        logger.debug('Cannot look at %s URL %s', vcs_scheme, link)
+        logger.warning('Cannot look at %s URL %s', vcs_scheme, link)
```

Comment:
We should make this message more specific (e.g. include the above comment to explain *why* pip ignores a VCS URL here).

Reply:
That's a good idea. I have included the above comment in message in https://github.com/pypa/pip/pull/8465/commits/dec8fe6f5257aafffa6e04665664227bd9d5b437